backend/src/neuroagent/tools/entitycore_memodel_getall.py

```diff
@@ -0,0 +1,116 @@
+"""Get all Me-Models."""
+
+import logging
+from typing import ClassVar
+from uuid import UUID
+
+from httpx import AsyncClient
+from pydantic import Field
+
+from neuroagent.tools.autogenerated_types.entitycore import (
+    ListResponseMEModelRead,
+    ReadManyMemodelGetParametersQuery,
+)
+from neuroagent.tools.base_tool import BaseTool, EntitycoreMetadata
+
+logger = logging.getLogger(__name__)
+
+
+class MeModelGetAllInput(ReadManyMemodelGetParametersQuery):
```

Comment:
I would call it `ME...` instead of `Me` - to be consistent with the meaning + entitycore class naming